xarray.ipynb

```diff
@@ -0,0 +1,361 @@
+    "is useful to gain insight on the computation.  \n",
+    "\n",
+    "The link to the dashboard will become visible when you create the client below.  We recommend having it open on one side of your screen while using your notebook on the other side.  This can take some effort to arrange your windows, but seeing them both at the same is very useful when learning."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client, progress\n",
+    "client = Client(n_workers=2, threads_per_worker=2, memory_limit='1GB')\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Open a sample dataset\n",
+    "\n",
+    "We will use some of xarray's tutorial data for this example. All we need to do is specify the chunk shape and xarray will automatically create Dask arrays for the variables in the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "\n",
+    "ds = xr.tutorial.load_dataset('air_temperature',\n",
+    "                              chunks={'lat': 25, 'lon': 25, 'time': 1000})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By specifying the chunk sizes by dimension name, we have configured the data variables in our `Dataset` to be returned backed by Dask arrays. In xarray, `Datasets` are dict-like container of labeled arrays akin to the `pandas.DataFrame`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Quickly inspecting the Dataset above, we'll note that this Dataset has three dimensions akin to axis in NumPy (lat, lon, and time), three coordinate variables akin to Pandas.Index objects (also named lat, lon, and time), and one data variable (air). Xarray also holds Dataset specific metadata in as Attributes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = ds['air']\n",
+    "da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each data variable in xarray is called a `DataArray`. These are the fundemental labeled array object in xarray. Much like the Dataset, DataArray's still have dimensions and coordinate that support many of its array opperations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Accessing the underlying array of data is done via the `data` property. Here we can see that we have a Dask array."
```

Comment:
It seems I don't get a dask-array, not sure why, but nothing shows up in Dask dashboard while trying the notebook.

See output of the cell above:
```
array([[[241.2    , 242.5    , 243.5    , ..., 232.79999, 235.5    ,
         238.59999],
        [243.79999, 244.5    , 244.7    , ..., 232.79999, 235.29999,
         239.29999],
        [250.     , 249.79999, 248.89   , ..., 233.2    , 236.39   ,
         241.7    ],
```

Reply:
So this was a bit of an odd thing on the xarray side. I've worked around it for now but we're considering changing some of the tutorial data loading behavior in xarray now: https://github.com/pydata/xarray/pull/2538